browser/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@0x/mesh-browser",
+    "version": "3.0.1-beta",
+    "description": "TypeScript and JavaScript bindings for running Mesh directly in the browser.",
+    "main": "./lib/index.js",
+    "license": "MIT",
+    "private": true,
```

Comment:
It's not private since we publish it to NPM.

Reply:
Yeah I just set it to private so I didn't accidentally publish it early. I'll change it back.